lib/apimatic-core/utilities/date_time_helper.rb

```diff
@@ -122,5 +122,52 @@ def self.from_rfc3339(date_time)
         DateTime.rfc3339("#{date_time}Z")
       end
     end
+
+    def self.validate_datetime(dt_format, dt)
+      case dt_format
+      when DateTimeFormat::HTTP_DATE_TIME
+        return DateTimeHelper.rfc_1123?(dt)
+      when DateTimeFormat::RFC3339_DATE_TIME
+        return DateTimeHelper.rfc_3339?(dt)
+      when DateTimeFormat::UNIX_DATE_TIME
+        return DateTimeHelper.unix_timestamp?(dt)
+      end
+
+      false
+    end
+
+    def self.validate_date(date_value)
```

Comment:
Nits, Non-blocker: This method named could be renamed to `valid_date?`.

Reply:
done